pkgs/development/python-modules/sphinx-autobuild/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, livereload
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-autobuild";
+  version = "2020.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vbaf4vrxahylyp8zrlw5dx1d2faajp926c3pl5i1wlkp1yll62b";
+  };
+
+  # No tests included.
+  doCheck = false;
+
+  propagatedBuildInputs = [ sphinx  livereload ];
```

Comment:
```suggestion
  propagatedBuildInputs = [ sphinx livereload ];
```

Reply:
Removed in 010ff4b7eebaa038e737b3a4e1535cf1479d5053